### PR TITLE
Story container to function component

### PR DESCRIPTION
**Pull request by FranceJ**
# Description

Updates the StoryContainer React component to be a function component instead of a class component.

## More Details

The associated test required a one line change due to a difference in how the component is being exported.

## Corresponding Issue

https://github.com/ifmeorg/ifme/issues/1656

# Screenshots

https://github.com/ifmeorg/ifme/issues/1656

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

File: client/app/components/BaseContainer/StoryContainer.jsx [hunk 1]
```diff
@@ -1,35 +1,35 @@
 // @flow
-import React from 'react';
+import React, { StatelessFunctionalComponent } from 'react';
```
Comment (julianguyen) [inline client/app/components/BaseContainer/StoryContainer.jsx:2]: You don't need to include `StatelessFunctionalComponent`!

File: client/app/components/BaseContainer/StoryContainer.jsx [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-              body={props.body}
-              medicationBody={props.medicationBody}
-            />
-          </div>
-        ))}
-      </div>
-    );
-  }
-}
+const StoryContainer = (props): StatelessFunctionalComponent<Props> => {
```
Comment (julianguyen) [inline client/app/components/BaseContainer/StoryContainer.jsx:10]: Here you can do `const StoryContainer = ({ data }): Props => {`

that way line 11 can be removed.

**Review by julianguyen — COMMENTED** [review]
Great work, thanks for working on this! Should be good to merge after the feedback is addressed :)

File: client/app/components/BaseContainer/StoryContainer.jsx (hunk 2 above)
Comment (FranceJ) [reply client/app/components/BaseContainer/StoryContainer.jsx:10]: good call, destructuring is great

**Review by julianguyen — APPROVED** [review]
Awesome work! Thanks for working on this, feel free to merge whenever!

**Pull request merged**